lametro/views.py

```diff
@@ -517,12 +518,21 @@ def dispatch(self, request, *args, **kwargs):
         return response
 
     def get_context_data(self, **kwargs):
-        post_model = LAMetroPost
 
         context = super().get_context_data(**kwargs)
         person = context['person']
 
-        context['qualifying_post'] = person.current_council_seat.post.acting_label
+        council_post = person.current_council_seat.post
+
+        context['qualifying_post'] = council_post.acting_label
+
+        if council_post.shape:
+            context['map_geojson'] = serialize('geojson',
```

Comment:
i'm tempted to also try a serialize solution for the boardmember map, but it's not so straightforward. Might think a bit more about that,